schemas/webpackOptionsSchema.json

```diff
@@ -76,7 +76,14 @@
         },
         {
           "additionalProperties": {
-            "type": "string"
+            "anyOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "object"
+              }
+            ]
           },
```

Comment:
Hmm, it seems that the error formatter doesn't handle this properly. I think we should continue using `anyOf` then, also because we'll probably need it when adding more specific properties.